### ISSUE TITLE
Select menus not accessible as expected

### ISSUE THREAD

**Issue by gavinorland** [formio/formio.js#3063]
Hi. We use Formio extensively and one issue we're seeing is that select menus are not functioning as expected with a screen reader:

**Expected  behaviour:**

Each time the user uses the keyboard to change the focused option, the screen reader should read out all the options again. If the user actually selects an option, the screen reader should read that one out.

**Actual behaviour:**

When the user uses the keyboard to change the focused option, the screen reader does not read anything. When the user actually selects an option, the screen reader does read that one out.

This is the case with JAWS and we think with Edge Read Aloud too.

Do you think there's much prospect of the expected SR support being added to these components?

**Comment by VikkiAlenn** [comment]
Closing this thread as it is outdated. Please re-open if it is still relevant. Thank you for your contribution!
